### ISSUE TITLE
Video cannot be played if not logged into a Google account

### ISSUE THREAD

**Issue by limzykenneth** [processing/hello-p5js-video#1]
@tuantinghuang 

The URLs used for the video on this [line](https://github.com/processing/hello-p5js-video/blob/da553a92d5c12854060b9346403436b27626d929/js/script.js#L5) points to a Google Cloud storage account reference. To be able to load and view the video, the user must have an active session logged into their Google account in their browser.

If possible it would be better to use a CDN service to serve the video instead, if it is easier to setup, [Google Cloud CDN](https://cloud.google.com/cdn) can do the job. Do let me know if any help is needed to set these up though.

**Comment by limzykenneth** [comment]
Is the video currently not on the foundation's account?

**Comment by tuantinghuang** [comment]
No, I didn't have access to the foundation's account, so I'm currently using the CDN from my school google account!

**Comment by limzykenneth** [comment]
@tuantinghuang I've moved the video to the foundation's Cloudflare account so we have storage and CDN there. Can you try swapping out for this URL and see if there is any problem? https://hello-assets.p5js.org/p5-video-draft-3.webm

Another thing is that I might want to try reencoding the video to reduce the file size, I'm not familiar with the player you are using however, it is possible for it to support multiple sources a la `<video>` tag with multiple `<source>` tags?

**Comment by tuantinghuang** [comment]
Sorry for the delay, these R2 bucket links work! Thank you, Kenneth! I will send an updated version of videos once I have more details fixed.
